modules/http/src/main/java/org/mule/module/http/internal/request/grizzly/TlsTransportCustomizer.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.http.internal.request.grizzly;
+
+import org.mule.transport.ssl.api.TlsContextFactory;
+
+import com.ning.http.client.providers.grizzly.TransportCustomizer;
+
+import org.glassfish.grizzly.filterchain.FilterChainBuilder;
+import org.glassfish.grizzly.nio.transport.TCPNIOTransport;
+import org.glassfish.grizzly.ssl.SSLFilter;
+
+/**
+ * Transport customizer that sets enabled cipher suites and protocols in the SSL filter.
+ */
+public class TlsTransportCustomizer implements TransportCustomizer
```

Comment:
I don't like this approach, but it's either this or extend (and duplicate a lot of code) from com.ning.http.client.providers.grizzly.GrizzlyAsyncHttpProvider.